### PR TITLE
ISPN-15067 All RESP handlers should use ASCII string whenever possible

### PR DESCRIPTION
**Pull request by wburns**
https://issues.redhat.com/browse/ISPN-15067

Also made some changes to prevent String concatenation for integer using more efficient write

### CODE REVIEW

File: server/resp/src/main/java/org/infinispan/server/resp/Resp3Handler.java [hunk 1]
```diff
@@ -122,6 +122,15 @@ public static void handleBulkResult(CharSequence result, ByteBufPool alloc) {
       }
    }
 
+   public static void handleBulkAsciiResult(CharSequence result, ByteBufPool alloc) {
+      if (result == null) {
+         handleNullResult(alloc);
+      } else {
+         ByteBufferUtils.stringToByteBufAscii("$" + result.length() + CRLF_STRING + result + CRLF_STRING,
+               alloc);
```
Comment (jabolina) [inline server/resp/src/main/java/org/infinispan/server/resp/Resp3Handler.java:130]: Should we write piece by piece instead?

**Review by jabolina — APPROVED** [review]
Just one question.

File: server/resp/src/main/java/org/infinispan/server/resp/Resp3Handler.java (hunk 1 above)
Comment (wburns) [reply server/resp/src/main/java/org/infinispan/server/resp/Resp3Handler.java:130]: Yes, we should. The breaking up I did in a separate pass and forgot about this and the handleBulkResult method.

**Comment by wburns** [comment]
Updated

**Comment by jabolina** [comment]
Thanks, @wburns!

**Pull request merged**